docs/transpile/custom_passes.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: Transpilation passes beyond the standard set
+description: Passes not included in the transpiler default optimization levels.
+
+---
+# Custom passes 
+
+Many transpiler passes within Qiskit are optional and are not executed in any of the default optimization levels. In some cases, they can take the place of default passes within those stages. Others can be written outside of Qiskit and executed as plugins. Here we introduce a pass, `PadDynamicalDecoupling`, that adds an error suppression technique called _dynamical decoupling_ to the circuit. Because this pass only operates on scheduled circuits and involves gates that are not necessarily basis gates of our `target`, we will need the `ALAPScheduleAnalysis` and `BasisTranslator` passes as well.
+
+First, import necessary libraries for building a circuit and working with passes. In particular, the `PassManager` is how we manage custom passes:
+
+```python
+    from qiskit import transpile
+    from qiskit.circuit.library import EfficientSU2, XGate, YGate
+
+    from qiskit.providers.fake_provider import FakeSherbrooke
+    from qiskit.transpiler import PassManager, InstructionProperties
+    from qiskit.transpiler.passes import BasisTranslator
+    from qiskit.transpiler.passes.scheduling import ALAPScheduleAnalysis, PadDynamicalDecoupling
+
+    import numpy as np
+```
+
+We will work with the backend `FakeSherbrooke`. We get the `target` information from the `backend` and save the operation names as `basis_gates` because we will need to modify the `target` to add timing information for the gates used in dynamical decoupling.
+
+```python
+    backend = FakeSherbrooke()
+
+    target = backend.target
+    basis_gates = list(target.operation_names)
+```
+
+Build an ansatz circuit to work with. Dynamical decoupling often works best when there is a lot of idle time in the quantum circuits. That is, there are qubits that are not being used while others are active. We can see this is the case in this circuit because the two-qubit `ecr` gates are applied sequentially in this ansatz.
+
+```python
+    qc = EfficientSU2(12, entanglement='circular', reps=1)
+    qc_t = transpile(qc, target=target, seed_transpiler=12345)
```

Comment:
General question - why is it necessary to run `transpile()` when you're going to be running your custom passes later on? Is there a specific reason to do 2 rounds of transpilation? I'm guessing im not the only one who would have this question so might be nice to add some explanation into the text itself :D

Reply:
I added a sentence of explanation:

> We'll first transpile the circuit to the backend because dynamical decoupling pulses need to be added after the circuit has been transpiled and scheduled.